### ISSUE TITLE
Data: Energy - Issues related to EPC data release

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-london#41]
As asked by @polly64 

https://www.owenboswarva.com/blog/post-hou2.htm records Owen Boswarva's detailed efforts to gain clarification on  permitted reuse of EPC data, from  Dept for levelling up, housing and communities (DLUHC),  and on the classification of EPC data as personal data subject to GDPR.

Though the outcome was that DLUHC revised view of EPC data as 'personal', in Colouring Cities it is considered an area where sensitivity is required as how much energy each household uses can be used to infer lifestyle, income, activities within tpersonal space. Because of this, though DLUHC EPC ratings will be recorded for each property in Colouring Britain (as they are published by government) they are not currently being visualised by building but aggregated at block level.

Aggregation is also necessary as a buildings may contain mutliple flats  each with different EPCs ratings. It is important note that EPC assessments are not carried out by trained surveyors.

**Comment by polly64** [comment]
-Also can add some EPC building age data to 'Architectural Style 'section ( ask Loughborough to prepare)  but only able to be done for certain age bands-(EPC age bands are: Pre1900; 1900-1929; 1930-1949; 1950-1966; 1967-1975; 1976-1982; 1983-1990; 1991-1995; 1996-2002; 2003-2006; 2007-2011; 2012 onwards). The ones that can be absorbed into the 'architectural style' section are as follows:
For 'Victorian'- Add 'Before 1900' band - (limited no. of Georgian buildings included in this can be extracted later using Listed building data)
For '1946-79 (Post war)' add 1950-1966 and 1967-1975 bands
For '1980-1999 (Late 20thc)' add  1983-1990 and 1991-1995 bands
For '2000-2025 (Early 21stc)' add 2003-2006; 2007-2011 & '2012 onwards' bands 
Others bands cannot be integrated as these cross over style dates but quite a lot of data will be able to be visualised using the above

<img width="440" alt="Screenshot 2023-08-28 at 23 44 22" src="https://github.com/colouring-cities/colouring-london/assets/42236514/21329841-34e6-4aed-bedf-c06160a89572">

**Comment by polly64** [comment]
@MKIM2008  to go through with PH and close

**Comment by MKIM1008** [comment]
https://github.com/colouring-cities/colouring-london/issues/57#issuecomment-1828868492

**Comment by polly64** [comment]
@MKIM1008 can you copy and paste all the text here , apart from the image showing periods, to the other energy issue so they are all together - and then close this issue thanks

**Comment by MKIM1008** [comment]
From #57 
**********************
M_Kim: Show any open SME or land use open Datasets for Loughborough available ideally mapped using GIS.
M_Kim & TL: Are we mapping land use using MSE data, or land use and SME information?
If both need to create new SME data class. How will this be updated?
I've discovered an SMEs dataset on Gov.uk and will bring it along this Wednesday for our review. This includes address information which enables us to map using GIS. * We can designate certain business types as land use categories.
PH: We could easily create an additional box which says tick if SME but should also have the date next to it and have options to add new

https://www.gov.uk/government/publications/companies-house-foi-responses-january-to-june-2022/a-list-of-small-and-medium-sized-enterprises-sme-in-the-uk
http://download.companieshouse.gov.uk/en_output.html

Regarding land use, are we able to use the data from DEC and EPC? If so, we should discuss how to effectively translate the land use codes to align with the current Colouring London's land use legend.
image
Or can we create a new DEC / EPC (land use) legend? This opinion is from the meeting with Falli.

PH: Yes definitely can extract land use and age data and excellent to do, Grant and Joe at Birmingham have already worked in age I know

*There's not enough land use data in OSM and I will review it more.

PH: OSM data can be supplemented and corrected using the crowdsourcing feature - we need to make sure the date is added as this is the fastest changing of all our datasets. This will also enable land use history and land use change to be tracked from now which is very useful.

Once we have age and land use data we can use these to infer other characterisatics such as materials and look at probability of accuracy. We can talk to experts in our international group in terms of possibly testing AI and ML approaches on the data as they may well be interested.

M_Kim and TL, Does the current Colouring Loughborough 'mixed use' feature under 'Land Use' also require
an option for floor level for each different land use to be put in?
I've noticed that in the DEC and EPC data, land use is categorised as 'mixed-use' for entire buildings, without specifying the land use for each individual floor. I'll look into whether there's more detailed information available on this matter.

PH: Thanks - I think providing evidence that retail nearly always will fall on ground floor for example is iactually mportant especially in inference work - ie what are the rules that we can use - this will help with rapid generation of data for Charnwood as a whole to give the regional context for Charnwood and also promote work on the regional prototype. I am going to write this up in the manual too
